scripts/update_op_coverage.py

```diff
@@ -62,7 +63,8 @@ def get_backend_op_pair(test):
   test_suite_backends = {
       'check_vmla_vmla': 'vmla',
       'check_llvm-ir_llvm': 'llvm-ir',
-      'check_vulkan-spirv_vulkan': 'vulkan-spirv'
+      'check_vulkan-spirv_vulkan': 'vulkan-spirv',
+      'check_dylib-llvm-aot_dylib': 'dylib-llvm-aot',
```

Comment:
We could instead make the `raise LookupError(f'Can not find a backend to match {test}')` check less opinionated.